src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Integrate a Flutter module into your iOS project
+short-title: Integrate Flutter
+description: Learn how to integrate a Flutter module into your existing iOS project.
+---
+
+TODO(jmagman): add the pre-built framework alternative and review.
+
+## Integration steps
+
+To integrate a Flutter module into your app, use the following steps:
+
+ 1. [Create Flutter Module](#create-a-flutter-module)
+    Creates the Flutter project which is hosted in the existing app.
+ 1. [Add Module Dependency in Existing
+    App](#make-the-host-app-depend-on-the-flutter-module)
+    Sets up XCode to include your Flutter project into the build of your
+    existing app.
+
+## Create a Flutter module
+
+Flutter projects created using `flutter create xxx` include simple host apps for
+your Flutter/Dart code (a single-ViewController iOS host). You can modify these
+host apps to suit your needs and build from there.
+
+If you're starting off with an *existing* host app for either platform, you'll
+likely want to include your Flutter project in that app as some form of library
+instead.
+
+In order to embed Flutter like a library, use the Flutter module template.
```

Comment:
"like a library" => "as you would a library"

Reply:
changed to "as a library"